oss-cloud-backend/services/databaseService.js

```diff
@@ -39,9 +39,9 @@ module.exports.getContributorsPaging = (params) => mysql
     [params.searchParam, params.searchParam, params.searchParam,
       params.sortBy === undefined ? 'username' : params.sortBy,
       (params.page - 1) * params.itemsPerPage, params.itemsPerPage],
-  )
-  .query(`SELECT COUNT(*) FROM contributors WHERE INSTR(username, ?) > 0 
-    OR INSTR(firstName, ?) > 0  OR INSTR(lastName, ?) > 0 `,
+  ) // add show hidden :)

```

Comment:
Is this a TODO comment?

Reply:
It was a TODO comment, but we forgot to remove it after we had done it.
It is removed now.